src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java

```diff
@@ -259,7 +268,9 @@ public DidChangeTextDocumentParams getChangesParams() {
      * @param c The character just typed
      */
     public void characterTyped(char c) {
-        // Todo - Implement
+        if (signatureTriggers.contains(Character.toString(c))) {
```

Comment:
As i can see, this code expects one-symbol string, cause `c` is a char.
@NipunaRanasinghe

Reply:
so... it could be converted to List<Character> i guess

Reply:
Yeah that is one option :) We can either convert the List<String> to List<Character> when reading the server options or we can keep it as it is and do the check like this.. Just a matter of where the conversion should be done right? Which one do you prefer?

Reply:
@NipunaRanasinghe please, add ` around your Lists :D

Reply:
@nixel2007 oh my bad :smile: done!